packages/salesforcedx-vscode-lightning/syntaxes/html.tmLanguage.json

```diff
@@ -0,0 +1,746 @@
+{
```

Comment:
Is this use completely as-is or did we have to modify some things? Asking to see if there is anything I need to focus on.

Reply:
I modified nothing about it, only took the previous version of it which doesn't contain the rules that check for invalid tags.